src/ray/core_worker/tests/shutdown_coordinator_test.cc

```diff
@@ -280,28 +280,6 @@ TEST_F(ShutdownCoordinatorTest, Worker_HandleExit_OnIdleTimeout) {
                                            ShutdownReason::kIdleTimeout));
 }
 
-TEST_F(ShutdownCoordinatorTest, ShouldEarlyExit_Performance_IsFast) {
-  auto coordinator = CreateCoordinator();
-  auto start = std::chrono::high_resolution_clock::now();
-  constexpr int iterations = 1000000;
-  volatile bool result = false;
-
-  for (int i = 0; i < iterations; ++i) {
-    result = coordinator->ShouldEarlyExit();
-  }
-
-  auto end = std::chrono::high_resolution_clock::now();
-  auto duration = std::chrono::duration_cast<std::chrono::nanoseconds>(end - start);
-
-  // Should be very fast (less than 100ns per call on modern hardware)
-  double ns_per_call = static_cast<double>(duration.count()) / iterations;
-  EXPECT_LT(ns_per_call, 100.0)
-      << "ShouldEarlyExit too slow: " << ns_per_call << "ns per call";
```

Comment:
we should not write this type of test in CI as a general rule; it's destined to be flaky.

if we want to measure performance, we should do it as a release test where we actually track the metrics over time.

testing this specific call also doesn't make much sense to me.